packages/functional-tests/tests/settings/changeEmail.spec.ts

```diff
@@ -0,0 +1,162 @@
+    expect(await login.signInError()).toMatch(
+      'Primary account email required for sign-in'
+    );
+
+    // Success signing in with New email
+    await login.useDifferentAccountLink();
+    await login.login(newEmail, credentials.password);
+    const primary = await settings.primaryEmail.statusText();
+    expect(primary).toEqual(newEmail);
+  });
+
+  test('change primary email, password and login', async ({
+    credentials,
+    page,
+    pages: { settings, changePassword, login },
+  }) => {
+    const newPassword = credentials.password + '@@2';
+    await settings.password.clickChange();
+    await changePassword.setCurrentPassword(credentials.password);
+    await changePassword.setNewPassword(newPassword);
+    await changePassword.setConfirmPassword(newPassword);
+    await changePassword.submit();
+    await settings.signOut();
+
+    // Sign in with old password
+    await login.setEmail(newEmail);
+    await page.locator('button[type=submit]').click();
+    await login.setPassword(credentials.password);
+    await page.locator('button[type=submit]').click();
+    expect(await login.signInPasswordTooltip()).toMatch('Incorrect password');
+
+    // Sign in with new password
+    credentials.password = newPassword;
+    await login.setPassword(credentials.password);
+    await login.submit();
+    const primaryEmail = await settings.primaryEmail.statusText();
+    expect(primaryEmail).toEqual(newEmail);
+  });
+
+  test('change primary email, change password, login, change email and login', async ({
+    credentials,
+    pages: { settings, changePassword, login },
+  }) => {
+    const newPassword = credentials.password + '@@4';
+    await settings.password.clickChange();
+    await changePassword.setCurrentPassword(credentials.password);
+    await changePassword.setNewPassword(newPassword);
+    await changePassword.setConfirmPassword(newPassword);
+    await changePassword.submit();
+    await settings.signOut();
+
+    // Sign in with new password
+    credentials.password = newPassword;
+    await login.login(newEmail, credentials.password);
+
+    // Change back the primary email again
+    await settings.secondaryEmail.clickMakePrimary();
+    await settings.signOut();
+
+    // Login with primary email and new password
+    await login.login(credentials.email, credentials.password);
+    const primaryEmail = await settings.primaryEmail.statusText();
+    expect(primaryEmail).toEqual(credentials.email);
+  });
+
+  test('can change primary email, delete account', async ({
+    credentials,
+    page,
+    pages: { settings, deleteAccount, login, secondaryEmail },
+  }) => {
+    // Click delete account
+    await settings.clickDeleteAccount();
+    await deleteAccount.checkAllBoxes();
+    await deleteAccount.clickContinue();
+
+    // Enter the correct password
+    await deleteAccount.setPassword(credentials.password);
+    await deleteAccount.submit();
+    await page.pause();
```

Comment:
Q: WHat's this for?
https://playwright.dev/docs/api/class-page#page-pause

Reply:
oops! that was added for debugging. I will remove it in this [PR](https://github.com/mozilla/fxa/pull/14132)
Thanks for catching that! :)